spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/users/:user_id/posts/'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'returns the appropriate placeholders' do
+      get '/users/:user_id/posts'
+      expect(response.body).to include('Here is a list of posts for a given user')
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http success' do
+      get '/users/:user_id/posts/show'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'returns the appropriate placeholders' do
+      get '/users/:user_id/posts/show'
+
+      expected_content = 'This is a particular post with the given ID out of all ' \
+                         'the posts for a given user'
+
+      expect(response.body).to include(expected_content)
+    end
+  end
+end
```

Comment:
- You have done well so far, but according to the project requirements, you are also required to test if the right template was rendered. For example `expect(response).to render_template(:show)` to test if it renders the `post#show` page. Please fix this for all test including `users_spec` 🙏

Reply:
I appreciate you @mckent05. Thank you so much